external/fv3fit/fv3fit/emulation/thermobasis/emulator.py

```diff
@@ -252,7 +259,9 @@ def step(self, in_, out):
         return info
 
     def get_loss(self, in_, out):
-        return self.config.target.loss(self.model(in_), out)
+        prediction = self.model(in_)
+        prediction_loss, info = self.config.target.loss(prediction, out)
+        return prediction_loss + sum(self.model.losses), info
```

Comment:
What this function does has changed but the name didn't - could you add a docstring describing this behavior?